sleap/info/write_tracking_h5.py

```diff
@@ -252,6 +260,7 @@ def main(labels: Labels, output_path: str, all_frames: bool = True):
     data_dict = dict(
         track_names=track_names,
         node_names=get_nodes_as_np_strings(labels),
+        edge_names=get_edges_as_np_strings(labels),
```

Comment:
Can we also save `skeleton.edge_inds` as a separate dataset (`/edge_inds`)? It's just an integer `(n_edges, 2)` array. It's redundant with `edge_names`, but it's a lot more convenient in some applications (e.g., plotting).

Reply:
Yes, `edge_inds` have been added. Although, I wish I had more carefully looked at the skeleton attributes before using the `edge_names` attribute for the bento plugin. Anyway, having both `edge_names` and `edge_names` will be good for verification puposes.